flocker/acceptance/testtools.py

```diff
@@ -117,7 +117,8 @@ def get_node_state(node):
     return FlockerConfiguration(state).applications()
 
 
-def _run_SSH(port, user, node, command, input, key=None):
+def _run_SSH(port, user, node, command, input, key=None,
+             background=False):
```

Comment:
`background` is undocumented and definitely not obvious what it does from the name :smile: Good thing we're going to get rid of all this SSH stuff soon.